frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -87,3 +90,20 @@ export const detectOCSSupportedFeatures: FeatureDetector = async (dispatch) => {
       : handleError(error, _.keys(OCS_SUPPORT_FLAGS), dispatch, detectOCSSupportedFeatures);
   }
 };
+
+export const detectLSO: FeatureDetector = async (dispatch) => {
+  try {
+    const subscription = await fetchK8s<K8sResourceKind>(
+      SubscriptionModel,
+      'local-storage-operator',
+      LSO_NAMESPACE,
+    );
+    await fetchK8s(ClusterServiceVersionModel, subscription?.status?.currentCSV, LSO_NAMESPACE);
+
+    dispatch(setFlag(LSO_FLAG, true));
+  } catch (err) {
+    err?.response?.status !== 404
+      ? handleError(err, [LSO_FLAG], dispatch, detectLSO)
+      : dispatch(setFlag(LSO_FLAG, false));
+  }
+};
```

Comment:
This will run only once. If in middle the user would install LSO until the user refreshes the page console will not detect it. I suggest you don't use flags but just perform this fetch in the Install Page and accordingly show messages.